.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,11 +7,26 @@ assignees: ''
 
 ---
 
-**Description:** A clear and concise description of what the bug is.
+**Description:**
+A clear and concise description of what the bug is.
 
-**Expected behavior:** A description of what you expected to happen.
+**Expected behavior:**
+A description of what you expected to happen.
 
-**Actual behavior:** A description of what is actually happening.
+**Actual behavior:**
+A description of what is actually happening.
+
+**Task version:**
+Specify the task version
+
+**Platform:**
+- [ ] Ubuntu
+- [ ] macOS
+- [ ] Windows
```

Comment:
Is exact OS version important in some cases? Should we split it to Ubuntu 16.04, Ubuntu 18.04 and so on?

Reply:
I don't think that it makes sense to split it.